.gitignore

```diff
@@ -0,0 +1,6 @@
+/.phpcs-cache
+/composer.lock
+/phpcs.xml
+/phpstan.neon
```

Comment:
PHP static analysis tools haven't usually been up to the leve of Python ones, what is the judgment on phpstan so far?

Reply:
Ok so far, some big(ger) projects are using it (eg Doctrine). It's at the highest level (since this is so small), so can reduce if it goes weird.